models/issues/comment.go

```diff
@@ -130,6 +130,12 @@ const (
 	CommentTypePRScheduledToAutoMerge
 	// 35 pr was un scheduled to auto merge when checks succeed
 	CommentTypePRUnScheduledToAutoMerge
+	// 36
+	CommentTypeCloseAsArchived
+	// 37
+	CommentTypeCloseAsResolved
+	// 38
+	CommentTypeCloseAsMerged
```

Comment:
I guess it's better to extend the `CommentTypeClose` , instead of adding too many new comment types.